### PR TITLE
Milestone 3

### PR DESCRIPTION
**Pull request by alexandermorales-dev**
Created  the remaining sections of original website

### CODE REVIEW

**Review by MkrtichSargsyan — CHANGES_REQUESTED** [review]
## Hello Guys


 You have done a lot in this project  :+1: .However, there are some changes that you need to make in order to get your project approved

Note that you have two more reviews on this project before you reach the review limit, so make all of the required changes before submitting for a review

  - [x] Kindly fix the horizontal scrollbar for small sizes

  - [x] Don't use ```!important``` in the css files

  - [x] header is too big (reduce padding size for header)
  
  - [x] Change the font colors for the navigation
  
  ###  Your's
yours-font
  ### Original
original-font

## In the magazine and Editor's pick section

- [x] ```in the magazine```  content is not in the right place

### Your's
yours-magazine
### Original
magazine

 - [x] The first image for mobile size looks smaller
 
 ### Your's
 
mag-y

### Original
mag-o

- [x] Each title uses a hx tag and have a link (instead of ```p``` use ```hx``` tag)

## Main newspaper sections (US, World, etc.) section

- [x] Each image have a link

- [x] Fix image overlapping for small sizes

 ### Your's
 
overlap

## Subscribe / Newsletter section

- [x] Add a grey background to subscribe section

- [x] Fix image overlapping for mobile screen

 ### Your's
 
yours


### Original

original-sub





  


<hr>

## OPTIONAL

- [x] Add h1 tag to the page

  theoretically and logically wrong to have subheadlines ```<h2>``` without a main headline ```<h1>```,   but it won't have a negative effect.

- [x] Footer section has too big title for the mobile screen
### Original
footer
### Your's

footer-y



Please, make the REQUIRED changes and submit your request again! :grinning:
Good luck. 

> ps: . If you have any questions about my review or need help, you can message me on slack ( @mkrtichsargsyan )

File: style.css [hunk 1]
```diff
@@ -187,4 +402,8 @@ header {
     font-size: 1.5rem;
     color: #f4f5f6a9;
   }
+
+  .img-text-container {
+    display: block !important;
+  }
```
Comment (MkrtichSargsyan) [inline style.css]: Don't use ```!important``` in the css files

File: style.css [hunk 2]
```diff
@@ -132,51 +381,17 @@ header {
   }
 
   .pro-fa {
-    display: none;
+    display: none !important;
```
Comment (MkrtichSargsyan) [inline style.css]: Don't use ```!important``` in the css files

Comment (DanSam5K) [reply style.css]: Wow! Noted sir, we will try and Implement all the bugs you spotted to make the required changes.👍

Comment (MkrtichSargsyan) [reply style.css]: write me on slack if you have any question or need a help

**Review by teekaytech — APPROVED** [review]
# Status: Approved :green_circle: 

### Congratulations team :tada: :clinking_glasses: 

_**Thank you for fixing all the changes required by the last reviewer :handshake: **_

**This project has passed all the mandatory requirements :clap: 
It is therefore approved :heavy_check_mark:**

**However, it will be nice if you can check my comments below and implement them before you merge. I'm marking them as optional because they're not raised by the first reviewer. I strongly suggest you fix them.**

**Comment(s)**
- In mobile view sizes, kindly centralize the contents of the `In the magazine`, `Editor's pick`, and the `Main newspaper (US, World, etc.)` sections. 
- Also on mobile view, your subscribe section layout is a bit different from the original. Check the images below for the difference. 

  | Original Page | Your Page |
  | ------------------ | --------------|
  | original | your page |

Again, congratulations, you can merge to the default branch.
Do not forget to fill the project completion form, as this is important for your weekly progress score.

**_Good luck with your next task._**

* * * 
**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Comment by DanSam5K** [comment]
Duly Noted Sir!🤝

**Pull request merged**